workflow/controller/operator.go

```diff
@@ -2643,7 +2643,7 @@ func (woc *wfOperationCtx) processAggregateNodeOutputs(tmpl *wfv1.Template, scop
 			resultsList = append(resultsList, item)
 		}
 	}
-	if tmpl.GetType() == wfv1.TemplateTypeScript {
+	{
```

Comment:
This was (in v2.12) including container templates. I don't see any reason to guard this at all, as anything step that could produce it could be used.